spec/parser/srt_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+describe Subber::Parser::Srt do
+  describe '.parse' do
+    let(:content) { read_fixture('srt-sample.srt') }
+
+    it 'generates subtitles based on the file content' do
+      subtitles = described_class.parse(content)
+      expect(subtitles).to_not be_empty
+
+      subtitle1, subtitle2 = subtitles
```

Comment:
Looping through the array of subtitles will help prevent false positive test result

```rb
expected_attribute_hashes = []

subtitles.each_with_index do |subtitle, index|
  attribute_hash = expected_attributes_hashes[index]
  expect(subtitle).to have_attributes(attribute_hash)
end
```